src/src.builds

```diff
@@ -2,9 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectExclusions Condition="'$(OS)' != 'Windows_NT'" Include="$(MSBuildThisFileDirectory)**\src\System.Memory.csproj" />
```

Comment:
No other projects are specifically disabled like this (or if they are, then they have been that way, via .builds files, for a long time).  I think both ways of disabling are reasonable. Disabling via configurations means one less piece, but disabling via excludes makes it a more intentional change.  My preference is to do it via configurations because I think it's cleaner, but that requires someone to be very aware of what is / isn't / should / shouldn't be building.